workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectComponentDispatcherBase.cs

```diff
@@ -24,20 +24,20 @@ internal abstract class GameObjectComponentDispatcherBase
         public ComponentGroup[] CommandRequestsComponentGroups { get; set; }
 
         public abstract void MarkComponentsAddedForActivation
-            (Dictionary<int, MonoBehaviourActivationManager> entityIndexToManagers);
+            (Dictionary<Entity, MonoBehaviourActivationManager> entityIndexToManagers);
```

Comment:
Can  you rename the argument name `entityIndexToManagers` to something like `entityToManager`?